docker-bake.hcl

```diff
@@ -32,34 +32,16 @@ target "dynbinary" {
     }
 }
 
-variable "GROUP_TOTAL" {
-    default = "1"
-}
-
-variable "GROUP_INDEX" {
-    default = "0"
-}
-
-function "platforms" {
-    params = [USE_GLIBC]
-    result = concat(["linux/amd64", "linux/386", "linux/arm64", "linux/arm", "linux/ppc64le", "linux/s390x", "darwin/amd64", "darwin/arm64", "windows/amd64", "windows/arm", "windows/386"], USE_GLIBC!=""?[]:["windows/arm64"])
-}
-
-function "glen" {
```

Comment:
n00b question; 😅 these are all no longer needed? Why were they needed before?

Reply:
They were needed for CircleCI `cross.parallelism` which provides [`CIRCLE_NODE_INDEX` and `CIRCLE_NODE_TOTAL` env vars](https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables) to handle efficient parallelism.

Reply:
Ah! Gotcha